Update @typescript-eslint/{parser,eslint-plugin} peer/dev dependencies to ^6.0.0

toastwaffle opened the pull request.

sweepline commented:
Did you test this? I guess we might also have to bump this package a major version? I'm currently between vacations, so help would be appreciated.

toastwaffle commented:
I've tested this using `npm run test`, and by installing it from a tarball in my own project

I've changed the version bump to be a major version (and updated the readme to explain)

solcik commented:
I can confirm, that your plugin is working well in my quite big project after upgrade to v6 of @typescript-eslint/{parser,eslint-plugin}
